scout/core/templates/case.html

```diff
@@ -78,7 +90,7 @@
 
       {% if case.status != 'archived' %}
         <input type="hidden" name="status" readonly value="archived">
-        <button class="case-assign-icon" type="submit" onclick="return confirm('Are you sure?')">
+        <button class="case-assign-icon" type="submit" onclick="return confirm('Are you sure? This will disable the alignment view and delete analysis files. You will have to request a FULL rerun to continue evaluating e.g. research variants.')">
```

Comment:
I had to laugh when reading this. I hope no-one found out the hard way what archiving means.